etc/indico.conf.sample

```diff
@@ -300,8 +300,8 @@ MaxUploadFileSize = 0
 # OAUTH PARAMETERS
 #------------------------------------------------------------------------------
 
-# Time to live of the OAuth Tokens
-OAuthAccessTokenTTL = 10000
+# Time to live of OAuth grant tokens (in seconds)
+OAuthGrantTokenTTL = 100
```

Comment:
In case some Indico admin would want to make it very short or very long for whichever reason, but I think going for a sensible default would be enough.